### PR TITLE
Eventlet fix and some other stuff

### LINKED ISSUE #49: eventlet tests fail with "RuntimeError: no suitable implementation for this system"
```
File "/home/travis/build/ionelmc/python-manhole/.tox/2.7-signalfd-eventlet-nocov-patchthread/lib/python2.7/site-packages/eventlet/support/monotonic.py", line 167, in <module>
    raise RuntimeError('no suitable implementation for this system')
RuntimeError: no suitable implementation for this system
```

https://travis-ci.org/ionelmc/python-manhole/builds/319810334

### PR DESCRIPTION
**Pull request by ionelmc**
Fixes #49

### CODE REVIEW

File: src/manhole/__init__.py [hunk 1]
```diff
@@ -400,7 +400,7 @@ def __call__(self, message):
                     os.write(self.destination, full_message.encode('ascii', 'ignore'))
                 else:
                     self.destination.write(full_message)
-            except:  # pylint: disable=W0702
+            except Exception:  # pylint: disable=W0702
```
Comment (nirs) [inline src/manhole/__init__.py]: We can remove now the # pylint: comment:
http://pylint-messages.wikidot.com/messages:w0702

Is this related to the eventlet fix?

File: tests/helper.py [hunk 2]
```diff
@@ -41,6 +41,7 @@ def setup_greenthreads(patch_threads=False):
 
     try:
         import eventlet
+        eventlet.hubs.get_hub()
```
Comment (nirs) [inline tests/helper.py]: Why not add a comment why this is needed, linking to the eventlet patch adding this?

**Comment by nirs** [comment]
All tests pass now except this:

```
=========================== short test summary info ============================
FAIL tests/test_manhole.py::test_uwsgi
=================================== FAILURES ===================================
__________________________________ test_uwsgi __________________________________
tests/test_manhole.py:556: in test_uwsgi
    assert_manhole_running(proc, '/tmp/manhole-%s' % pid, oneshot=True)
tests/test_manhole.py:50: in assert_manhole_running
    sock = connect_to_manhole(uds_path)
tests/test_manhole.py:39: in connect_to_manhole
    sock.connect(uds_path)
E   FileNotFoundError: [Errno 2] No such file or directory
----------------------------- Captured stdout call -----------------------------
```

File: src/manhole/__init__.py (hunk 1 above)
Comment (ionelmc) [reply src/manhole/__init__.py]: Not really but `tox -e check` was complaining.

**Comment by ionelmc** [comment]
You get that failure locally? Which tox env?

**Comment by nirs** [comment]
No, copied from travis build.

**Comment by nirs** [comment]
@ionelmc, would you like to merge this?

**Comment by ionelmc** [comment]
About the flaky tests - I haven't been able to figure them out yet. Locally everything is silky smooth ...
